docs/galaxy/docsite/README.md

```diff
@@ -0,0 +1,48 @@
+Homepage and Documentation Source for Ansible Galaxy
+====================================================
+
+This project hosts the source behind [docs.ansible.com/galaxy](https://docs.ansible.com/galaxy)
+
+Contributions to the documentation are welcome. To make changes, submit a pull request that changes the reStructuredText files in the `rst/` directory only, and the core team can do a docs build and push the static files.
+
+If you wish to verify output from the markup such as link references, you may install sphinx and build the documentation by running `make webdocs` from the `ansible/docsgalaxy/docsite` directory.
```

Comment:
I know you just copied this from the main doc, so changes should probably go there . . . but I'm noting them down while I'm thinking of them. 

Can we add a link to Sphinx to the words "install sphinx"?
Also, there's a slash missing in the directory path (docsgalaxy).